src/ui/shared/requests/useAddressActivity.ts

```diff
@@ -1,14 +1,45 @@
-import { createDomainHook } from 'defi-sdk';
+import type { CachePolicy } from 'defi-sdk';
+import { createDomainHook, client } from 'defi-sdk';
 
 const namespace = 'address';
 const scope = 'activity';
```

Comment:
This is a deprecated endpoint. For now the best way is to use address/positions and check the result.

Reply:
Since when is it deprecated?! We rely on it in the import flow

Reply:
Hm, for quite a long time.
Let's ask backend